setup.py

```diff
@@ -55,9 +55,23 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    with open("./include/vrv/vrvdef.h") as header_file:
+        defines = {}
+        for line in header_file:
+            if not line.startswith("#define"):
```

Comment:
You can be very specific about what you're looking for if you use a simple regex, rather than parsing the whole file and /or shortcutting at a specific line:

```
# find a needle in a haystack. :)
import re
needle = re.compile(r"VERSION_MAJOR|VERSION_MINOR|VERSION_REVISION|VERSION_DEV")
...

    for line in header_file:
        if not needle.search(line):
            continue
...
```

This also means you don't have to tie it to a specific order -- if (for some reason) `vrv_cast` moves above the version declaration, then this will stop functioning. Since the `re` module is built-in to Python it doesn't add any new dependencies, and the speed is comparable.